brayns/common/types.h

```diff
@@ -297,8 +301,8 @@ enum class CameraMode
  * in absolute value of the coordinate system. Values are stored
  * in a Vector4, with the following order: nx, ny, nz and d
  */
-using ClipPlane = std::array<double, 4>;
-using ClipPlanes = std::vector<ClipPlane>;
+using Plane = std::array<double, 4>;
```

Comment:
Why the rename? `ClipPlane` still makes more sense.

Reply:
Because ClipPlane is now a different type (a Plane + an ID)